### DISCUSSION TITLE
Unable to view account discussions

### DISCUSSION THREAD

**Discussion by wes-kay** [community/community#38194] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm trying to access my discussions https://github.com/discussions but whenever I do I get a 500 server error: 
![image](https://user-images.githubusercontent.com/40011470/199776234-5a83be5e-84fd-4846-b638-27c8e4bd7300.png)

**Comment by david-engelmann** [comment]
[I had a similar discussion yesterday](https://github.com/community/community/discussions/38094). They are working on those endpoints. try orgs/community if community/community is broken

**Reply by wes-kay** [reply]
This is not for community discussions this is for account discussions. 
![image](https://user-images.githubusercontent.com/40011470/199781002-5e4fa173-8d47-4835-bf1d-739f402aa7d3.png)

**Comment by airtower-luna** [comment, marked as answer]
It's working for me right now, if you have a persistent error you might want to [send a bug report to support](https://support.github.com/contact/bug-report) so they can take a look.
